packageregistry/pypi.go

```diff
@@ -148,3 +152,7 @@ func (np *pypiPackageDiscovery) GetPackage(packageName string) (*Package, error)
 
 	return &pkg, nil
 }
+
+func (np *pypiPackageDiscovery) GetPackageDownloadStats(packageName string) (DownloadStats, error) {
+	return DownloadStats{}, nil
```

Comment:
Return error please ie. download stats is not supported, otherwise it will cause bugs in client code assuming it to be `0` downloads